src/azure-cli/azure/cli/command_modules/sql/commands.py

```diff
@@ -573,6 +573,7 @@ def load_command_table(self, _):
         g.show_command('show', 'get')
         g.custom_command('list', 'managed_instance_list')
         g.generic_update_command('update', custom_func_name='managed_instance_update', supports_no_wait=True)
+        g.command('failover', 'failover', supports_no_wait=True)
```

Comment:
how can we query failover status when add --no-wait?

Reply:
I guess by sending 
```
GET https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/DejanDuVnetRG/providers/Microsoft.Sql/locations/westus/managedInstanceAzureAsyncOperation/098ac4ab-d57b-40f1-9fb2-d8533df163f9?api-version=2019-06-01-preview
```
, you just need `requestId`.

Reply:
Is there a command or SDK method to query failover status?